plugins/Marketplace/vue/src/PluginDetails/PluginDetails.vue

```diff
@@ -0,0 +1,448 @@
+    },
+    isMatomoPlugin() {
+      return ['piwik', 'matomo-org'].includes(this.plugin.owner);
+    },
+    showReviews() {
+      return this.pluginReviews && this.pluginReviews.embedUrl;
+    },
+    showMissingLicenseDescription() {
+      return this.hasSomeAdminAccess && typeof this.plugin.isMissingLicense !== 'undefined' && this.plugin.isMissingLicense;
+    },
+    showExceededLicenceDescription() {
+      return this.hasSomeAdminAccess && typeof this.plugin.hasExceededLicense !== 'undefined' && this.plugin.hasExceededLicense;
+    },
+    showMissingRequirementsNoticeIfApplicable() {
+      return this.isSuperUser && (this.plugin.isDownloadable || this.plugin.isInstalled);
+    },
+    showLicenceName() {
+      return this.pluginLatestVersion
+        && this.pluginLatestVersion.license
+        && this.pluginLatestVersion.license.name;
+    },
+    showPluginVariations() {
+      return (!this.plugin.isDownloadable || !this.isSuperUser)
+        && !this.plugin.isEligibleForFreeTrial
+        && this.plugin.isPaid
+        && this.pluginShop
+        && this.pluginShopVariations.length;
+    },
+    pluginScreenshots() {
+      return this.plugin.screenshots || [];
+    },
+  },
+  methods: {
+    applyTabs() {
+      const root = this.$refs.root as HTMLElement;
+
+      setTimeout(() => {
+        $('#pluginDetailsTabs .tabs', root).tabs();
+      });
+    },
+    applySelect() {
+      setTimeout(() => {
+        const $variationPicker = $('.pluginDetails .variationPicker select');
+        if ($variationPicker.val()) {
+          $('.addToCartLink').attr('href', $variationPicker.val());
+        }
+        $variationPicker.on('change', () => {
+          $('.addToCartLink').attr('href', $variationPicker.val());
+        });
+
+        if ($variationPicker.length) {
+          $variationPicker.material_select();
+        }
+      });
+    },
+    getProtocolAndDomain(url: string) {
+      const urlObj = new URL(url);
+      return `${urlObj.protocol}//${urlObj.hostname}`;
+    },
+    applyIframeResize() {
+      setTimeout(() => {
+        const iFrameResize = window.iFrameResize || null;
+        if (this.pluginReviews && iFrameResize) {
+          $(() => {
+            const $iFrames = $('.pluginDetails iframe.reviewIframe');
+            for (let i = 0; i < $iFrames.length; i += 1) {
+              // eslint-disable-next-line max-len
+              iFrameResize({ checkOrigin: [this.getProtocolAndDomain(this.pluginReviews.embedUrl)] }, $iFrames[i]);
+            }
+          });
+        }
+      });
+    },
+    sha1(content: string) {
+      const hash = crypto.createHash('sha1');
+      hash.update(content);
+      return hash.digest('hex');
+    },
+    getScreenshotBaseName(screenshot: string) {
+      const filename = screenshot.split('/').pop() || '';
+      return filename.substring(0, filename.lastIndexOf('.')).split('_').join(' ');
+    },
+    showPluginDetailsDialog() {
+      $('#pluginDetails').modal({
```

Comment:
What browser was that in? It was working ok for me in Brave browser.

Reply:
The screenshots were from a current desktop Chrome. My current Firefox has less issues with the scrollbars, but the vertical overflow on the right column is also visible there at like 800px viewport height.

Reply:
I'm on a Mac so can't test other operating systems (Linux, Windows) so would appreciate any cooperation on this :)

Reply:
It looks like `#pluginDetails { overflow-y: initial }` is part of the problem. Removing it will make the complete modal scrollable, probably an undesired state.

But with `overflow: hidden` we face the issue of the parts itself not being scrollable. Throwing around `height: 100%` on `.modal_content, .pluginDetails, .row, .col` (with an additional `overflow-y: scroll` on `.col`) made it partially scrollable, yet still horrible, because the `<h2>` outside of `.pluginDetails` is messing with the total space available and pushes things down.

In my local browsers it looks like we have to override everything to flexbox so much that it may be easier to just remove `.row / .col` and build it fresh.

Reply:
In that case it might be best to spend the time and energy and refactor the markup into the final state as per the phase 2 designs, I don't think refactoring the markup and styles to achieve the same is the best use of the time.